plugin/tower/pkg/informer/reflector.go

```diff
@@ -323,8 +339,26 @@ func (t *gqlType) ListName() string {
 }
 
 // QueryFields return the type fields as gql query fields.
-func (t *gqlType) QueryFields() string {
-	return utils.GqlTypeMarshal(t, true)
+func (t *gqlType) QueryFields(skipFields map[string]string) string {
+	return utils.GqlTypeMarshal(t, skipFields, true)
+}
+
+// matchFieldNotExistFromMessage matchs field which not exist from error message.
+// It returns two string values, the first is the field name, the second is the parent name.
+// A return value of nil indicates no match.
+func matchFieldNotExistFromMessage(message string) []string {
```

Comment:
If the message format or content updated, this may cause other issue, right?

Reply:
These changes are to compatibility with the old version of tower. If the tower API is forward compatibility, this error won't occur.